frontend/ui/widget/menu.lua

```diff
@@ -971,7 +971,11 @@ function Menu:updatePageInfo(select_number)
             self.page_info_text:setText(FFIUtil.template(_("Page %1 of %2"), self.page, self.page_num))
             self.page_info_text:enable()
         else
-            self.page_info_text:setText("");
+            self.page_info_text:setText("Single page");
+            -- Dirty hackery: Call "disable" to to neuter its input handler...
+            self.page_info_text:disable()
+            -- ...and then fudge the label manually to make sure the text is black...
+            self.page_info_text.label_widget.fgcolor = Blitbuffer.COLOR_BLACK
```

Comment:
Make that hackery a method of Button ? `Button:disableButLookLikeEnabled()` :/ ?

Reply:
It might be enabled for one-page as well, no harm. To simplify the code.